Automattic/wp-calypso
# Update/form discussion settings

In this PR, I'm trying to reduxify another Settings Form in order to see how we can factorize the behaviors of all these settings form and use the last coding guidelines. I'm trying to answer the question: How should we replace the `formBase` mixin.

Edit: The proposed solution is using a HoC for all the settings forms

**Testing instructions**

 - Navigate to the discussion settings page
 - Try to update and refresh all the settings
 - the settings should be updated correctly
 - try switching between sites while on this page
